Frontend/src/pages/homepage/Homepage.scss

```diff
@@ -0,0 +1,24 @@
+
+.homepage{
```

Comment:
please research how to prettify a React project using chat GPT and prettify all files of this PR

Reply:
oops, my bad :)